lib/widgets/message_list.dart

```diff
@@ -453,6 +460,12 @@ class _UnreadMarker extends StatelessWidget {
   final bool isRead;
   final Widget child;
 
+  // The color hsl(227deg 78% 59%) comes from the Figma mockup at:
+  //   https://www.figma.com/file/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=132-9684
+  // See discussion about design at:
+  //   https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/flutter.3A.20unread.20marker/near/1658008
+  static Color color = const HSLColor.fromAHSL(1, 227, 0.78, 0.59).toColor();
```

Comment:
```suggestion
  static final color = const HSLColor.fromAHSL(1, 227, 0.78, 0.59).toColor();
```

That way it's clear this is meant to function as a constant, and not to ever get mutated.